src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1315,15 +1346,79 @@ void Graph::InferDynamic(SyncInferRequest* request) {
 }
 
 inline void Graph::ExecuteNode(const NodePtr& node, const dnnl::stream& stream) const {
-    DUMP(node, getConfig().debugCaps, infer_count);
-
-    OV_ITT_SCOPED_TASK(itt::domains::intel_cpu, node->profiling.execute);
-    DEBUG_LOG(*node);
-    if (node->isDynamicNode()) {
-        node->executeDynamic(stream);
+    if (!node->parallelWith.empty()) {
+        // run nodes in parallel
+        auto& parallelNodes = node->parallelWith;
+        if (node == parallelNodes[0]) {
+            if (const auto& cpuExecutor = context->getCPUStreamExecutor()) {
+                auto num_parallel_nodes = parallelNodes.size();
+                ParalleMtNuma(num_parallel_nodes, cpuExecutor, [&](int subStreamID, size_t i) {
+                    auto& n = parallelNodes[i];
+
+                    n->toNumaNode(subStreamID);
```

Comment:
@sunxiaoxia2022 @xczhai subStreamID may be different from numa node id. Should we change as below or the logic is handled in other place?
```suggestion
                    n->toNumaNode(cpuExecutor->get_numa_node_id(););
```